### PR TITLE
tic

### PR DESCRIPTION
**Pull request by maelle**
Expected to address #4 #5 #85 

What's missing are the custom tic.R templates for training, analysis and basic projects.

* not sure what should be in a basic project tic.R

* in training and analysis the idea will "simply" be to knit things from a folder and store them somewhere (that folder? a branch?) but I'm stuck because the directories can be set by the user, so it's hard to know which one has the Rmd to knit. Thoughts @stephlocke?

### CODE REVIEW

**Comment by stephlocke** [comment]
I think what I'd prefer to do is generate the tic and if their layout matches the default then we add stuff, otherwise we add a comment saying INSERT LOGIC HERE.

**Comment by maelle** [comment]
And does the rest of he infrastructure added in this PR look ok?

File: R/createPackageProject.R [hunk 1]
```diff
@@ -142,13 +143,19 @@ createPackageProject <- function(name, title = NULL,
...
+            coverage = coverage
           )
         }
 
         # README
         knit_readme()
 
+        # dummy test
+        usethis::use_testthat()
+        usethis::use_test(name = "dummy", open = FALSE)
```
Comment (stephlocke) [inline R/createPackageProject.R]: Dummy may not be a word someone wants in their commit history
```suggestion
        usethis::use_test(name = "sample", open = FALSE)
```

File: R/use_readme.R [hunk 2]
```diff
@@ -2,6 +2,7 @@ use_readme <- function(name){
   usethis::use_template("README.Rmd",
                         data = list(Package = name),
                         package = "starters")
+  usethis::use_build_ignore("README.Rmd")
```
Comment (stephlocke) [inline R/use_readme.R]: Does there also need to be .md build ignore added?

Comment (maelle) [reply R/use_readme.R]: I think some folks like to submit the README to CRAN but maybe that's not the way 🤔

Comment (stephlocke) [reply R/use_readme.R]: Fair enough then :)

**Comment by maelle** [comment]
I almost created a helper function to knit all Rmd in a directory, for use in tic.R, before removing it because I do not want projects to depend on `starters` (package projects have `starters` in Suggests because of the `get_project_health()` function but that doesn't make me happy).

**Comment by maelle** [comment]
@stephlocke current questions

* What folder to knit reports of for training projects?

* I've added the tic.R for analysis projects. I haven't added conditions on folders present because if there's no "analysis" folder then there's no report there so nothing happens. `tic.R` has comments. Is this fine?

**Comment by maelle** [comment]
I think `rmarkdown::render` Will automatically render the right format based on YAML info. Will finalize this PR  tomorrow 🎉

**Pull request merged**